File: ai.py
import os
import openai

openai.api_key = os.getenv("OPENAI_API_KEY")
response = openai.Completion.create(
    engine="gpt-4o-mini",
    prompt="AI says:",
    max_tokens=10
)
print("Python:", response.choices[0].text.strip())

File: ai.js
const { Configuration, OpenAIApi } = require("openai");

const configuration = new Configuration({
  apiKey: process.env.OPENAI_API_KEY,
});
const openai = new OpenAIApi(configuration);

(async () => {
  const res = await openai.createCompletion({
    model: "gpt-4o-mini",
    prompt: "AI says:",
    max_tokens: 10,
  });
  console.log("JavaScript:", res.data.choices[0].text.trim());
})();